addons/website_slides/models/slide_channel.py

```diff
@@ -45,8 +45,18 @@ def _compute_completion(self):
             slide_done = mapped_data.get(record.channel_id.id, dict()).get(record.partner_id.id, 0)
             slide_total = channel_data.get(record.channel_id.id) or 1
             record.completion = math.ceil(100.0 * slide_done / slide_total)
-            if record.completion >= 100:
-                record.update({'completed': True})
+
+    @api.multi
+    def _write(self, vals):
+        if 'completion' in vals and vals['completion'] >= 100:
+            vals.update({'completed': True})
```

Comment:
comment to indicate we want to force completed=True when completion is 100%
in a docstring, to explain this is because completion is computed stored, and completed should be set manually normally.

Reply:
Should be in its own commit actually because this is a fix (+ remove update just above)